libsabledb/src/commands/generic_commands.rs

```diff
@@ -170,6 +174,34 @@ impl GenericCommands {
         };
         Ok(Some(md.value_type()))
     }
+
+    async fn expire(
+        client_state: Rc<ClientState>,
+        command: Rc<RedisCommand>,
+        response_buffer: &mut BytesMut,
+    ) -> Result<(), SableError> {
+        let builder = RespBuilderV2::default();
+        // at least 2 arguments
+        check_args_count!(command, 2, response_buffer);
+
+        let db_id = client_state.database_id();
+
+        // EXPIRE key seconds
+        let key = command_arg_at!(command, 1);
+        let timeout = command_arg_at!(command, 2);
+
+        let generic_db = GenericDb::with_storage(&client_state.store, db_id);
+        if !generic_db.contains(key)? {
+            builder.null_string(response_buffer);
+        }
+
+        let mut reader = U8ArrayReader::with_buffer(&timeout);
```

Comment:
No need to deal with this, just create a `GenericDb` and call `get_expiration` and `put_expiration` afterwards to update